config/coordinator/coordinator-docker-traces-v2-override.config.toml

```diff
@@ -18,17 +18,20 @@ expected-traces-api-version-v2="v0.8.0-rc1"
 [traces.counters-v2]
 endpoints=["http://traces-node-v2:8545/"]
 request-limit-per-endpoint=20
-request-retry.max-retries=4
 request-retry.backoff-delay="PT1S"
 request-retry.failures-warning-threshold=2
 [traces.conflation-v2]
 endpoints=["http://traces-node-v2:8545/"]
 request-limit-per-endpoint=2
-request-retry.max-retries=4
 request-retry.backoff-delay="PT1S"
 request-retry.failures-warning-threshold=2
 
 [dynamic-gas-price-service]
 geth-gas-price-update-recipients=[
   "http://l2-node:8545/"
 ]
+
+[l2-network-gas-pricing.legacy.sample-transaction-gas-pricing]
+plain-transfer-cost-multiplier=1.0
+compressed-tx-size=350
+expected-gas=29400
```

Comment:
IIUC, `expected-gas` shall be deterministic so I suggest convention over configuration, and have a default in the code.

Reply:
I'm overriding it to something else. Default is 21000, which corresponds to a plain transfer, here I'm using something else

Reply:
I added a proper doc, that hopefully explains these numbers